controller/src/main/java/org/jboss/as/controller/CompositeOperationHandler.java

```diff
@@ -113,6 +113,7 @@ public void handleRollback(OperationContext context, ModelNode operation) {
                     ModelNode stepResponse = responseMap.get(stepName);
                     if (stepResponse.hasDefined(FAILURE_DESCRIPTION)) {
                         failureMsg.get(ControllerLogger.ROOT_LOGGER.compositeOperationFailed(), ControllerLogger.ROOT_LOGGER.operation(stepName)).set(stepResponse.get(FAILURE_DESCRIPTION));
+                        failureMsg.get(ControllerLogger.ROOT_LOGGER.compositeOperationFailed(), "Operation: " + ControllerLogger.ROOT_LOGGER.operation(stepName)).set(operationMap.get(stepName));
```

Comment:
Not right; this drops the failure details from the top level failure description and replaces them with the full details of the step operation, including params, which I don't really want to output. The format of the address (dmr syntax) isn't great either. If we output operation details it needs to be in addition to the failure info we already report.